### ISSUE TITLE
France TV authentification

### ISSUE THREAD

**Issue by Qedinux** [melmorabity/plugin.video.francetv#12]
France TV requiert une authentification pour accéder au contenu. Des paramètres permettant de s'authentifier personnellement seront prochainement très certainement nécessaires. Un développement pro-actif serait utile.

**Comment by melmorabity** [comment]
Bonjour,

y a-t-il eu des annonces en ce sens côté France.tv ?

**Comment by Qedinux** [comment]
Non. Je ne connais pas le rythme de développement, ni les objectifs de France.TV. Mais, il le feront certainement. Sinon, pourquoi auraient-ils mis en place une authentification ?

Il faudrait déjà prévoir un système de paramètre permettant d'introduire son identifiant et son mot de passe. Ensuite, une procédure de login afin d'obtenir le ou les token d'une session avec son identifiant. Finalement, lorsque ce sera requis, il faudra valider l'accès au contenu avec le ou les token obtenu.

J'ai souhaité ajouté le label "enhancement" mais je n'y suis pas parvenu car je pense ne pas en avoir pas le droit.
